config/order.toml

```diff
@@ -1,5 +1,6 @@
 [raft]
 batch_timeout               = "0.3s"  # Block packaging time period.
+rebroadcast_timeout         = "3m" # Node timeout tick to check if there exist txs to be rebroadcast
```

Comment:
Please change it to ‘tolerance_time’ to be consistent with the rbft configuration naming.